src/app/components/shared/menu/internal-link/internal-link.component.spec.ts

```diff
@@ -47,6 +48,7 @@ describe("MenuInternalLinkComponent", () => {
         [placement]="placement"
         [tooltip]="tooltip"
         [route]="route"
+        [qsp]="qsp"
```

Comment:
Maybe a better name would be `[params]`? Maybe investigate to check that the RouterLink directive doesn't interfere with using `[queryParams]`